.gitignore

```diff
@@ -27,4 +27,4 @@ out
 *.sublime-project
 *.sublime-workspace
 build-local.properties
-
+.checkstyle
```

Comment:
.checkstyle file is generated by the eclipse I am using in my current dev environment. I think I installed the checkstyle plugin in my eclipse at some point. I added .checkstyle to the .gitignore to avoid accidental checkins of those files.